azure-simple/Packer/Vagrantfile

```diff
@@ -0,0 +1,40 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
@rasanjaya85 please update the license header year. There are other applicable files as well.